### DISCUSSION TITLE
Page like `docs` for viewing statically hosted resources

### DISCUSSION THREAD

**Discussion by isConic** [fastapi/fastapi#7128] [Questions] [answered]
`/docs` is a neat way to view documentation of fast-api rest endpoints.  

---  

What if there were a URL endpoint to display our statically mounted drives?  
Maybe it would like this  

![image](https://user-images.githubusercontent.com/48657113/94738592-a3bea200-033d-11eb-9c86-242dbf40bc03.png)

---

**Comment by Kludex** [comment]
I think it can be a plugin, but I don't see why it needs to be in FastAPI itself.

Anyhow, I don't understand the purpose of this feature. Do you mind explaining the it?

**Comment by tiangolo** [comment, marked as answer]
I imagine it could be interesting for some use cases, but that would belong to another project, not in FastAPI.

If you (or anyone) ends up building it, you can add the GitHub topic `fastapi`, and it will show up in the section about external projects. 🚀 

Given that, I'll close this issue. But thanks for the interest!

> Sorry for the long delay! 🙈  I wanted to personally address each issue and they piled up through time, but now I'm checking each one in order.
